modules/authentication/proc-provisioning-users-from-microsoft-azure-to-the-software-catalog.adoc

```diff
@@ -0,0 +1,277 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-provisioning-users-from-microsoft-azure-to-the-software-catalog_{context}"]
+= Provisioning users from Microsoft Azure to the software catalog
+
+.Prerequisites
+* You xref:proc-adding-azure-as-an-authentication-provider_{context}[enabled authentication with Microsoft Azure].
+
+.Procedure
+
+* To enable Microsoft Azure member discovery, edit your custom Developer Hub config map, such as app-config-rhdh, and add following lines to the app-config-rhdh.yaml content:
++
+[id=microsoftGraphOrgProviderId]
+.`app-config.yaml` fragment with mandatory `microsoftGraphOrg` fields
+[source,yaml]
+----
+dangerouslyAllowSignInWithoutUserInCatalog: false # <1>
+catalog:
+  providers:
+    microsoftGraphOrg:
+      providerId:
+        target: https://graph.microsoft.com/v1.0
+        authority: https://login.microsoftonline.com
```

Comment:
Are the `target` and `authority` mandatory? Or optional? And why would a customer change their value?

Reply:
Target is mandatory while authority is optional (see [comments](https://github.com/backstage/backstage/blob/918d273bbb843bd09bb6c80d22d87c257a7e5343/plugins/catalog-backend-module-msgraph/config.d.ts#L31-L41)). From what I understand, target refer to the MSGraph API endpoint the provider is connecting to, the user can change this for different versions. For instance, there is the [beta endpoint](https://learn.microsoft.com/en-us/graph/api/overview?view=graph-rest-beta#call-the-beta-endpoint).

As for authority, seems like there's different "authorities" where the is an unique URL for specific use cases. ([national cloud](https://learn.microsoft.com/en-us/graph/deployments#app-registration-and-token-service-root-endpoints))